### PR TITLE
Implement `Display` for `MiscellaneousSelect`

### PR DESCRIPTION
**Pull request by samdealy**

### CODE REVIEW

**Comment by samdealy** [comment]
Current dependencies on/for this PR:
* main
  * **PR #308** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/308" target="_blank">Graphite</a> 
    * **PR #310** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/310" target="_blank">Graphite</a> 
      * **PR #311** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/311" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/311?utm_source=stack-comment).

File: core/types/src/attributes.rs [hunk 1]
```diff
@@ -121,10 +121,17 @@ bitflags! {
...
 pub struct MiscellaneousSelect(sgx_misc_select_t);
 
-impl_newtype! {
+impl_newtype_no_display! {
     MiscellaneousSelect, sgx_misc_select_t;
 }
 
+impl Display for MiscellaneousSelect {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        write!(f, "MiscellaneousSelect: ")?;
```
Comment (jcape) [inline core/types/src/attributes.rs:130]:
```suggestion
```

File: core/types/src/attributes.rs [hunk 2]
```diff
@@ -211,4 +218,15 @@ mod test {
...
+
+    #[test]
+    fn miscellaneous_select_display() {
+        let sgx_misc_select_t = 18983928;
+        let miscellaneous_select = MiscellaneousSelect::from(sgx_misc_select_t);
+
+        let display_string = format!("{}", miscellaneous_select);
+        let expected_string = format!("MiscellaneousSelect: 0121_ABF8");
+
+        assert_eq!(display_string, expected_string);
```
Comment (jcape) [inline core/types/src/attributes.rs:230]:
```suggestion
        let display_string = format!("{}", miscellaneous_select);
        let expected_string = format!("0x0121_ABF8");

        assert_eq!(display_string, expected_string);
```

**Review by jcape — CHANGES_REQUESTED** [review]

File: util/src/format.rs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022-2023 The MobileCoin Foundation
+#![deny(missing_docs, missing_debug_implementations, unsafe_code)]
+//! Contains format utilities for formatting Hex representations of integers and byte arrays.
+
+use core::fmt::Formatter;
+
+/// A helper method that displays a u32 as Hex with a "_" delimiter.
+pub fn fmt_hex(src: &[u8], f: &mut Formatter) -> core::fmt::Result {
```
[the remark this answers is not in the document]
Comment (samdealy) [reply util/src/format.rs:8]: Done.

File: core/types/src/attributes.rs (hunk 2 above)
Comment (samdealy) [reply core/types/src/attributes.rs]: Done.

**Review by jcape — APPROVED** [review]

**Pull request merged**